index.js

```diff
@@ -144,14 +144,20 @@ ShopifyToken.prototype.getAccessToken = function getAccessToken(shop, code, fn)
       body += chunk;
     });
     response.on('end', function end() {
+      var error;
       if (status !== 200) {
-        return fn(new Error('Failed to get Shopify access token, status: ' + status + ', ' + body));
+        error = new Error('Failed to get Shopify access token');
+        error.responseBody = body;
+        error.statusCode = status;
+        return fn(error);
       }
 
       try {
         body = JSON.parse(body);
       } catch (e) {
-        return fn(new Error('Failed to parse the response body (' + body + ')'));
+        error = new Error('Failed to parse the response body');
+        error.responseBody = body;
```

Comment:
Please also add the `statusCode`, it's `200 `and is probably redundant but I would like to have it for consistency.